src/GameplayHud.cpp

```diff
@@ -68,8 +70,31 @@ void GameplayHud::load() {
     scores_ = { score_p1, score_p2, score_p3, score_p4 };
 
     for (auto& score : scores_) {
-        score.scale(0.0f);
+        score.set_scale(0.0f);
+    }
+
+    float pointer_size_x = 0.08f;
+    float pointer_size_y = pointer_size_x * 16.f / 9.f;
+    std::array<glm::vec2, 4> player_screen_centers = {
+        glm::vec2(-0.5f - pointer_size_x / 2.f, 0.5f - pointer_size_y / 2.f),
+        glm::vec2(0.5f - pointer_size_x / 2.f, 0.5f - pointer_size_y / 2.f),
+        glm::vec2(-0.5f - pointer_size_x / 2.f, -0.5f - pointer_size_y / 2.f),
+        glm::vec2(0.5f - pointer_size_x / 2.f, -0.5f - pointer_size_y / 2)
```

Comment:
Might want to make these numbers a const somewhere so that it'd be easy to change later. Especially when we get to polishing our game in a few weeks.

Reply:
I'll comment instead.  I think making them consts would make it hard to read due to length.